notebooks/demo-train-with-faq.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<p>Do we want to leave this commented out?</p><p> </p><p>In any case, I believe the CSVs included in the PR should be updated to match the current output (all metrics)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>